buildSrc/src/main/groovy/web/BeagleWebFilter.groovy

```diff
@@ -0,0 +1,45 @@
+import org.apache.tools.ant.filters.BaseFilterReader
+import org.apache.commons.io.IOUtils 
+
+/**
+ * Abstract class to quickly create FilterReaders that operate on a whole file content string.
```

Comment:
'whole' → 'entire'

I've seen this a lot from you and @RomanLangrehr . http://english.stackexchange.com/questions/9615/whole-vs-entire